EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -30,16 +31,17 @@ public static string ComputePEPValuesForAllPSMsGeneric(List<PeptideSpectralMatch
             Dictionary<string, Dictionary<int, Tuple<double, double>>> fileSpecificTimeDependantHydrophobicityAverageAndDeviation_unmodified = new Dictionary<string, Dictionary<int, Tuple<double, double>>>();
             Dictionary<string, Dictionary<int, Tuple<double, double>>> fileSpecificTimeDependantHydrophobicityAverageAndDeviation_modified = new Dictionary<string, Dictionary<int, Tuple<double, double>>>();
 
-            if (searchType == "standard")
```

Comment:
This was previously a bottom-up only conditional (standard == not topdown), but now it can be used with TD? Can we calculate hydrophobicities for proteins?

Reply:
yes. works well for TD. We use hydrobphobicities only for bottom up data.